Source/CombatTrainingMod/CombatTrainingTracker.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System.Collections.Generic;
+using RimWorld;
+using Verse;
+
+namespace KriilMod_CD
+{
+	public static class CombatTrainingTracker
+	{
+        private static Dictionary<string, SkillXpValues> PawnMeleeSkillValues = new Dictionary<string, SkillXpValues>();
+        private static Dictionary<string, SkillXpValues> PawnShootingSkillValues = new Dictionary<string, SkillXpValues>();
```

Comment:
Do these dictionaries get loaded somehow when you restore from a save? As far as I can tell, they wouldn't be.

Reply:
Nope.  I believe @Borattata mentioned this in his comments. I might take look at integrating these into the save.